Issue #1235 - Escalation only notify users

DavidVadnais opened the pull request:
Fixes #1235. Allows users to add a flag to the escalation command to only notify users and not make a followup entry or increase priority.

I tested this in my local environment but environment is slowly pulling away from the default one.

DavidVadnais commented:
I am not sure were to add this feature in the docs.  Is [configuration page](https://django-helpdesk.readthedocs.io/en/v1.3.0/configuration.html) the correct page?

Benbb96 commented:
> I am not sure were to add this feature in the docs. Is [configuration page](https://django-helpdesk.readthedocs.io/en/v1.3.0/configuration.html) the correct page?

Yes that seems like the right place.

Something like  "Adding the flag `--notify-only` will only send the emails and no followup will be added to the ticket" will do I suppose

DavidVadnais commented:
Here is a screen grab of the docs after running `make html` in the docs directory. 

![2025-03-08_helpdesk-doc-configuration](https://github.com/user-attachments/assets/3a04fd1e-defc-4f65-8d4d-b4d84f824fa9)

Please let me know if this was the correct way to edit the documentation, as I’m not very familiar with rst.

Benbb96 commented:
Yes that's the correct way of doing it. I would just add backticks around the `--notify-only` flag. And IMO, there's no need to add another step. you could just put your explanation in the 4th point since it is about the same command.

uhurusurfa reviewed (dismissed):
👍  Looks good

DavidVadnais commented:
I moved the new field to bullet point four and added back-ticks around `--notify-only`. I removed the `--` from the sentence because when it rendered it appeared as one dash. I have attached a screenshot of the docs page after running `make html`. 

![2025-03-10_helpdesk-doc-configuration](https://github.com/user-attachments/assets/16cbe1e3-2a96-4c0f-983a-76a27e02c575)

Let me know if anything else is needed!

Benbb96 reviewed (approved):
It looks good to me, thanks for this new feature!
